commands/init.js

```diff
@@ -80,7 +82,7 @@ function createDockerCompose(dir) {
   // add a 'shell' process for persistent changes, one-off tasks
   processes.shell = _.extend(_.cloneDeep(processes.web), {
     command: 'bash',
-    volumes: ['.:/app/user']
+    volumes: ['.:' + path.join('/app/user',appDir)]
```

Comment:
Would it be useful to be able to specify something other than `/user`, so that by default the volume would mount at `/app/user` but you could, for instance, use `/app/src` for Go, or `/app/foo/bar`, etc?